### DISCUSSION TITLE
Multiple Segmentation with Bounding Box trouble

### DISCUSSION THREAD

**Discussion by motviybletb** [roboflow/notebooks#158] [Q&A]
I need to segment many objects in my picture, so I tried to segment many objects by boxes, but in result I see only one box was drawn first.
Do you know how to fix that?

**Comment by SkalskiP** [comment]
Could you give me a bit more background for that question? Do we talk about our notebook or in general?

**Reply by motviybletb** [reply]
Yes, we speak about notebook 'how-to-segment-anything-with-sam'
I'm interested in part 'Generate Segmentation with Bounding Box'
Using bboxwidget I segment area with object 
In example with dog was only one bbox, but I need much more objects 
Here is my picture: 
![image](https://github.com/roboflow/notebooks/assets/98744794/9849b1d4-d37d-4d9f-a737-95822de68dc2)
![image](https://github.com/roboflow/notebooks/assets/98744794/a35013e4-3e61-464a-93d3-d15af7de4cbb)

I segmented 2 cells, but as result got only one
How to fix that?

p.s. I think I can use a loop for every square, but maybe there is an easier way to solve it
